### PR TITLE
Navbar Merge

### PR DESCRIPTION
**Pull request by knarfammussa**
Navbar without window-resizing responsiveness/mobile compatibility

### CODE REVIEW

File: chessence-frontend/src/components/Navbar/navbar.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from "react"
+import "./navbar.css"
+
+export default function Navbar() {
+  useEffect (() => {
+    const imgElement = document.querySelector('img[alt="Chessence"]');
+      imgElement.width = 72;
```
Comment (Xeygy) [inline chessence-frontend/src/components/Navbar/navbar.js:7]: Why are you manually setting image width in javascript and not in css?

Comment (Xeygy) [reply chessence-frontend/src/components/Navbar/navbar.js:7]: I think it should be done by setting an id on the element (e.g. `Chessence`), then setting the width and height in the css (e.g. `#navLogo {width: 72px; height 72px;}`)

**Review by Xeygy — CHANGES_REQUESTED** [review]

File: chessence-frontend/src/components/Navbar/navbar.js (hunk 1 above)
Comment (knarfammussa) [reply chessence-frontend/src/components/Navbar/navbar.js]: > I think it should be done by setting an id on the element (e.g. `Chessence`), then setting the width and height in the css (e.g. `#navLogo {width: 72px; height 72px;}`)

Got it; thanks! Couldn't figure out the css code for this; turns out it came down to the "#" in the front lol. It's good for review now.

Comment (Xeygy) [reply chessence-frontend/src/components/Navbar/navbar.js]: Looks good!

**Review by Xeygy — APPROVED** [review]
Nice work

**Pull request merged**